executor/partition_table_test.go

```diff
@@ -222,6 +226,43 @@ func (s *partitionTableSuite) TestPartitionInfoDisable(c *C) {
 	tk.MustQuery("select * from t_info_null where (date = '2020-10-02' or date = '2020-10-06') and app = 'xxx' and media = '19003006'").Check(testkit.Rows())
 }
 
+func (s *partitionTableSuite) TestOrderByandLimit(c *C) {
```

Comment:
Please add this at the beginning of this function to skip it in DATA RACE mode:
```
	if israce.RaceEnabled {
		c.Skip("exhaustive types test, skip race test")
	}
```

Reply:
It's updated now. PTAL.